build.gradle

```diff
@@ -67,7 +67,6 @@ plugins {
     id 'nebula.ospackage' version '8.4.1'
     id 'org.sonarqube' version '3.3'
     id 'com.adarshr.test-logger' version '2.1.0'
-    id 'com.palantir.git-version' version '0.12.3'
     id 'com.moowork.node' version '1.3.1'
     id 'net.nemerosa.versioning' version '3.0.0'
```

Comment:
Can we also remove `net.nemerosa.versioning` like in the other file?

Reply:
Oops, missed that one.